### DISCUSSION TITLE
Enable OEM Camera on Orange Pi 5 Pro

### DISCUSSION THREAD

**Discussion by xSPARExSTEWx** [Joshua-Riek/ubuntu-rockchip#915] [Q&A]
Everything with this has been working on my orange pi 5 pro, but I can't seem to get the OEM cameras to work. I got the OV13855 version installed in both cam1 and cam2 ports, neither work.

I have tried updating u-boot with orangepi-5-pro.dtb device tree and orange-pi-5-pro-cam1.dtbo device tree overlay and that did not do anything.

Not really sure what else to try here.

**Comment by Joshua-Riek** [comment]
I just did a bunch of testing on this board and camera, did you install camera-engine-rkaiq-rk3588?

**Reply by xSPARExSTEWx** [reply]
Yes I installed that first and tried the device tree after.

**Reply by Joshua-Riek** [reply]
I mean, how are you interacting with the camera? I do know it works, but I do not have a lot of time right now. It's possible you are using the wrong commands.
